integration-tests/contracts/ValueCalls.sol

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: MIT
+// @unsupported: evm
+pragma solidity >=0.7.0;
+
+contract ValueContext {
+    function getSelfBalance() external view returns(uint256) {
```

Comment:
Based on your callout on the PR into my branch I added [this check](https://github.com/ethereum-optimism/optimism/blob/01a14f7a5914009a5e91fd2552c6f1c980a3f903/integration-tests/test/native-eth-ovm-calls.spec.ts#L98-L106) which is used in multiple tests, so that we are asserting all of `ovmSELFBALANCE`, `ovmBALANCE`, and `eth_getBalance` (RPC) in each case.

Reply:
Oh, maybe you just mean to explicitly show that it works on the compiler side of things as opposed to the opcodes?  That would make more sense, I can add.

Reply:
added.